test/toDoController.spec.js

```diff
@@ -0,0 +1,40 @@
+describe('ToDoController', function() {
+  beforeEach(module('ToDo'));
+
+  var scope, ctrl;
+
+  beforeEach(inject(function($rootScope, $controller) {
+    scope = $rootScope.$new();
+    ctrl = $controller('ToDoController', {
+      $scope: scope
+    });
+  }));
+
+  it('initialises with an empty entry field', function() {
+    expect(scope.newToDo).toBeUndefined();
+  });
+
+  it('initialises with an empty list', function() {
+    expect(scope.todos).toEqual([]);
+  });
+
+  it('displays added tasks to the list', function() {
+    scope.newToDo = "walk the dog";
+    scope.addToDo();
+    expect(scope.todos[0].title).toEqual("walk the dog");
+  });
+
+  var itemDone = {"title":"shopping", "done":true};
+
+  it('displays a ticked checkbox when task is done', function() {
```

Comment:
This test doesn't actually check that a checkbox is displayed. Perhaps the test should be it('should know if a To-Do item is complete or not).